lib/src/lib.rs

```diff
@@ -4,6 +4,7 @@
 #![feature(plugin, decl_macro)]
 #![feature(never_type)]
 #![feature(try_trait)]
+#![feature(transpose_result)]
```

Comment:
This is need for [`Result<Option<T>>::transpose`](https://doc.rust-lang.org/nightly/std/result/enum.Result.html#method.transpose) method

Reply:
Ideally we wouldn't use this.

Reply:
Removed its usage.